ocean/transects/compute_transects.py

```diff
@@ -32,12 +32,14 @@ def get_mask_short_names(mask):
   mask = mask.set_index(nTransects=['transectNames', 'shortNames'])
   return mask
 
-def compute_transport(timeavg, mesh, mask, name='Drake Passage'):
+def compute_transport(timeavg, mesh, mask, name='Drake Passage',output='transport.nc'):
   mesh = xr.open_dataset(mesh)
   mask = get_mask_short_names(xr.open_dataset(mask))
 
   if name.lower() == 'all':
-    transectList = mask.shortNames[1:].values
+    transectList = mask.shortNames[:].values
+    condition = transectList != "Atlantic Transec"
```

Comment:
I'm a little confused here, are there sometimes `"Atlantic Transec"` in the output?  I didn't see that in the file that was produced from the previous version of this script.  I think I'm missing something here.

Reply:
In the 18to6 transect file there is an Atlantic transect that didn't make sense to run over (it's for MOC) and do the computation, so I pulled it out.  But I don't think it is a big deal to leave it in.